hospitals/views.py

```diff
@@ -21,6 +22,37 @@ def index(request):
 	
 	return render(request, 'index.html', index_context)
 
+prefixes = '''
+	PREFIX : <http://example.org/>
+	PREFIX schema: <http://schema.org/>
+	PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+	PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+'''
+def infographics(request):
+
+	# Make query calls from utils.py, and extract them
```

Comment:
I think we should agree on what indentation to use lol, but it would work for now